versal2.mk

```diff
@@ -0,0 +1,167 @@
+################################################################################
+# Following variables defines how the NS_USER (Non Secure User - Client
+# Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
+# S_USER (Secure User - TA) are compiled
+################################################################################
+override COMPILE_NS_USER   := 64
+override COMPILE_NS_KERNEL := 64
+override COMPILE_S_USER    := 64
+override COMPILE_S_KERNEL  := 64
+
+# Network support related packages:
+BR2_PACKAGE_DHCPCD ?= y
+BR2_PACKAGE_ETHTOOL ?= y
+BR2_PACKAGE_XINETD ?= y
+
+# SSH Packages :
+BR2_PACKAGE_OPENSSH ?= y
+BR2_PACKAGE_OPENSSH_SERVER ?= y
+BR2_PACKAGE_OPENSSH_KEY_UTILS ?= y
+
+# Openssl binary
+BR2_PACKAGE_LIBOPENSSL_BIN ?= y
+
+PLATFORM = AMD Versal Gen 2
+OPTEE_OS_PLATFORM = versal2
+OPTEE_OS_COMMON_EXTRA_FLAGS = CFG_PKCS11_TA=y CFG_USER_TA_TARGET_pkcs11=ta_arm64 O=out/arm
+
+################################################################################
+# Paths to git projects and various binaries
+################################################################################
+TF_A_PATH		?= $(ROOT)/arm-trusted-firmware
+U-BOOT_PATH		?= $(ROOT)/u-boot-xlnx
+LINUX_PATH		?= $(ROOT)/linux-xlnx
+OPTEE_OS_PATH		?= $(ROOT)/optee_os
+
+include common.mk
+
+################################################################################
+# Targets
+################################################################################
+
+all: tfa optee-os dtbo u-boot linux buildroot
+clean: tfa-clean optee-os-clean u-boot-clean linux-clean buildroot-clean dtbo-clean collectartifacts-clean
+
+include toolchain.mk
+
+###############################################################################
+# Collect all artifacts to output directory
+###############################################################################
+
+OUTPUTDIR = ../$(OPTEE_OS_PLATFORM)_output
```

Comment:
A few other platforms call this `BINARIES_PATH`, with the value `$(ROOT)/out` or `$(ROOT)/out/bin`.
Can we use the same convention here?